toast-tk-dao/src/main/java/io/toast/tk/dao/domain/impl/common/IServiceFactory.java

```diff
@@ -0,0 +1,11 @@
+package io.toast.tk.dao.domain.impl.common;
+
+import com.google.inject.assistedinject.Assisted;
+
+import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
+
+public interface IServiceFactory<E extends AbstractMongoDaoService<?>> {
+	
+	E create(final @Assisted String dbName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)